### ISSUE TITLE
error in graph_associations.py line 210, ValueError: cannot convert float NaN to integer

### ISSUE THREAD

**Issue by cnblevins** [fbdesignpro/sweetviz#29]
Error thrown up during analyze(dataframe), right after :PAIRWISE DONE: and Creating Associations graph...
Traceback (most recent call last):

  File "<ipython-input-22-2151284dc186>", line 1, in <module>
    myreport = sv.analyze(df)

  File "C:\Users\cnble\anaconda37\lib\site-packages\sweetviz\sv_public.py", line 13, in analyze
    pairwise_analysis, feat_cfg)

  File "C:\Users\cnble\anaconda37\lib\site-packages\sweetviz\dataframe_report.py", line 246, in __init__
    self._association_graphs["all"] = GraphAssoc(self, "all", self._associations)

  File "C:\Users\cnble\anaconda37\lib\site-packages\sweetviz\graph_associations.py", line 165, in __init__
    f = corrplot(graph_data, dataframe_report)

  File "C:\Users\cnble\anaconda37\lib\site-packages\sweetviz\graph_associations.py", line 410, in corrplot
    dataframe_report = dataframe_report

  File "C:\Users\cnble\anaconda37\lib\site-packages\sweetviz\graph_associations.py", line 318, in heatmap
    cur_size[1] / 2, facecolor=value_to_color(color[index]),

  File "C:\Users\cnble\anaconda37\lib\site-packages\sweetviz\graph_associations.py", line 210, in value_to_color
    ind = int(val_position * (n_colors - 1)) # target index in the color palette

ValueError: cannot convert float NaN to integer

**Comment by fbdesignpro** [comment]
@cnblevins thank you for submitting this! This is the first time I see this one, is the data set you are running this on public? (It would be great to test locally to fix) If not I will try to create a workaround.

**Comment by cnblevins** [comment]
Here's the dataset. There are some NaNs in the data, but I don't think that's what is throwing up this particular error.
Besides, I did a fillna(0) on the dataframe before calling sweetviz.

[JuneData.zip](https://github.com/fbdesignpro/sweetviz/files/4926795/JuneData.zip)

**Comment by cnblevins** [comment]
And, here is the code used (very straightforward), which threw up the error:
`import pandas as pd
import sweetviz as sv
df = pd.read_csv('C:/Users/cnble/.spyder-py3/JuneData.csv')
df.fillna(0)
test_report = sv.analyze(df)`

**Comment by a246530** [comment]
Recreated and error matches description by @cnblevins

**Comment by fbdesignpro** [comment]
@cnblevins Thank you! I will review this ASAP.

**Comment by fbdesignpro** [comment]
@cnblevins Thank you again for the report, it was caused by NaN values in the Numerical correlation. I have committed a stopgap workaround, which I hope to improve in future versions. But for now it should work, it will just show "no correlation" (0.0) if there is an error in the correlation (i.e. no valid data to do a correlation with). As I said I want to improve the display to explicitly show this fact in the future.
